.github/workflows/gpt-commit-summarizer.yml

```diff
@@ -0,0 +1,20 @@
+name: GPT Commits summarizer
+# Summary: This action will write a comment about every commit in a pull request, 
+# as well as generate a summary for every file that was modified and add it to the
+# review page, compile a PR summary from all commit summaries and file diff 
+# summaries, and delete outdated code review comments
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  summarize:
+    runs-on: ubuntu-latest
+    permissions: write-all  # Some repositories need this line
```

Comment:
This seems pretty sketchy...
Anyone want to weigh in?

Reply:
Specifically talking about it needing write-all permissions.

Reply:
Good catch. I'll update the PR later today.